src/megachatapi.h

```diff
@@ -4660,6 +4663,33 @@ class MegaChatApi
      */
     int getMaxVideoCallParticipants();
 
+    /**
+     * @brief Returns if audio level monitor is enabled
+     *
+     * It's false by default
+     *
+     * @note If there isn't a call in that chatroom in which user is participating,
+     * audio Level monitor will be always false
+     *
+     * @param chatid MegaChatHandle that identifies the chat room from we want know if audio level monitor is disabled
+     * @return true if audio level monitor is enabled
+     */
+    bool isAudioLevelMonitorEnabled(MegaChatHandle chatid);
+
+    /**
+     * @brief Enable or disable audio level monitor
+     *
+     * It's false by default and it's app responsability to enable it
```

Comment:
Missing documentation about the request and its params :)

Reply:
Fixed